region/norfair/west/Crocomire Speedway.json

```diff
@@ -329,14 +426,64 @@
         {"notable": "Reverse Spark"},
         "canShinechargeMovement",
         "canHorizontalShinespark",
-        {"shinespark": {"frames": 86, "excessFrames": 10}},
-        {"heatFrames": 700}
+        {"or": [
+          {"shinespark": {"frames": 87, "excessFrames": 10}},
+          {"and": [
+            "h_heatProof",
+            "canControlShinesparkEnd",
+            {"shinespark": {"frames": 87, "excessFrames": 44}}
+          ]}
+        ]},
+        {"heatFrames": 675},
+        {"or": [
+          "canSpeedball",
+          {"heatFrames": 15}
+        ]}
       ],
       "clearsObstacles": ["A"],
       "flashSuitChecked": true,
       "note": [
         "Spark left through the speed blocks, then run to the right and back to get speed to go through the rest.",
-        "The shinespark expects to kill the crumble bridge pirate, to be safe."
+        "If performing the spark with low energy, there is a risk of ending the spark above the spikes or in front of the Pirate;",
+        "to be safe, assuming Samus has heat protection and begins sparking about 3 tiles from the Speed blocks, start with between 72 and 87 energy or at least 105 energy;",
+        "with a buffered crumble jump to the right, up to 93 energy can work."
```

Comment:
This is missing canShinesparkMidair 🤔 And the difficulty of the midair spark scales with your leniency frames until you are doing a horizontal spark.  And I'd always try to standing spark this if I could.

Reply:
Yeah, at the amount of shinecharge frames (35) written in the strat, it would need `canMidAirShinespark`. But as long as you have enough leniency frames then it's not needed. Even at Very Hard, the leniency frames would be enough to ensure you could do a standing spark (as long as those frames weren't used in the other room). So I think it works out ok?

With the mid-air spark requirement being unwritten, the strat could be in Medium, except that the notable requirement puts it in Hard. It's probably for the best, because on Medium players would probably have trouble getting tripped up on the stairs.